client/src/components/ProjectSamplesTable.js

```diff
@@ -23,15 +25,21 @@ export const ProjectSamplesTable = ({
   // We only want to show the applied donwload options.
   // Also need some helpers for presentation.
   const { modality, format, getFoundFile } = useDownloadOptionsContext()
+  const { metadataComputedFile } = useMetadataOnly(project.computed_files)
   const [loaded, setLoaded] = useState(false)
   const [samples, setSamples] = useState(defaultSamples)
   const [showDownloadOptions, setShowDownloadOptions] = useState(false)
+  const [currentFilter, setCurrentFilter] = useState('') // For storing globalFiter for metadata only download
   const hasMultiplexedData = project.has_multiplexed_data
   const infoText =
     project && project.has_bulk_rna_seq
       ? 'Bulk RNA-seq data available only when you download the entire project'
       : false
 
+  const onFilterChange = (value) => {
+    setCurrentFilter(value)
```

Comment:
```suggestion
    setHasFilter(value !== '')
```

Reply:
See the commit at fc9f72d11bca05ea4211ef131cacc3fa3233113a.